src/wallet-web-api-swagger/Main.hs

```diff
@@ -83,7 +79,10 @@ instance ToSchema      BackupPhrase
 -- We need this instance for correct Swagger-specification.
 instance {-# OVERLAPPING #-} (Typeable a, ToSchema a) => ToSchema (Either WalletError a) where
     declareNamedSchema proxy = genericDeclareNamedSchema defaultSchemaOptions proxy
-        & mapped . name ?~ toText ("Either WalletError " ++ show (typeOf (undefined :: a)))
+        & mapped . name ?~ toText ("Either WalletError " ++ show (typeRep right))
```

Comment:
``` haskell
mapped . name ?~ show (typeRep (Proxy @(Either WalletError a)))
```

Or at the very least don't use `toText`, it's not needed here. `show` can produce `Text`.